### DISCUSSION TITLE
Rich text from a SharePoint list column will not be reflected as rich text within the panel content section

### DISCUSSION THREAD

**Discussion by jbmueller0584** [microsoft-search/pnp-modern-search#4084] [General]
Hi guys,

i do need help with this issue that i am having. Any help is highly appreciated!

I have a multiple lines of text column (set to rich text) in a SharePoint list. Here we added rich text in terms of underlined words and bullet point lists etc.

![2024-11-05 17_32_52-Escalation cases - Diverse Störungen, die zu Linienstillständen führen können un](https://github.com/user-attachments/assets/4b637724-8136-443a-8546-cd34f3821404)

The list column ist mapped with the managed property RefinableString171. I cant see any HTML formatting in the value of the managed property in the debug view, just some spaces.

![2024-11-05 17_40_39-Escalation Cases und 26 weitere Seiten - JBM – Microsoft​ Edge](https://github.com/user-attachments/assets/c8696352-7ef7-4d7c-8f3f-30f0e923c13d)

When i now want to display the value of that column / managed property within the panel to the right, rich text wont be reflected, and further more formatted as complete plain text.

![2024-11-05 17_36_55-Escalation Cases und 26 weitere Seiten - JBM – Microsoft​ Edge](https://github.com/user-attachments/assets/673368df-39f2-4166-85cc-429fe948bd55)

I am using a custom HTML template and tried to display with this manner {{{ }}} and also this manner {{ }}

![2024-11-05 18_05_17-eska-result-template-de html - Visual Studio Code](https://github.com/user-attachments/assets/355cef2a-34ac-4174-a350-0880a3b014cc)

Can you guys please help me out why the rich text is cutted down hardly to plain text? I saw in the past somewhere that it will work with rich text in general, but i really dont know what i am doing wrong therefore.

System that i am using: SharePoint Online Modern Communication site + PnP Modern Search 4.13.1

Thank you so much for your help!

**Comment by kasperbolarsen** [comment]
I added a site column using Extended RTF and added some content
In the extention SP Editor I now see this:
![image](https://github.com/user-attachments/assets/067c4a84-7d29-4f58-b82a-27db5eef36c8)


Can this solve your issue?

**Reply by jbmueller0584** [reply]
Hi Kasper Larsen, thank you for your reply. What is the "extention SP Editor"? And how you set up the "Extended RTF" format to the site column? via the UI or via PowerShell?

**Reply by kasperbolarsen** [reply]
SP Editor is a Chrome/Edge browser extension, you can read more about it here: https://github.com/pnp/sp-editor

As I recall it I set the Extended RTF in the UI
